com.clashcross.clashcross.yml

```
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+app-id: com.clashcross.clashcross
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: clashcross
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.login1
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json
+  # ClashCross
+  - name: ClashCross
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - mkdir -p clashcross
+      - tar -xf clashcross-linux-x86_64.tar.gz -C clashcross
+
+      # Copy the portable app to the Flatpak-based location.
+      - cp -r clashcross /app/
+      - mkdir /app/clashcross/clashcross
+      - chmod +x /app/clashcross/clashcross
+      - mkdir -p /app/bin
+      - ln -s /app/clashcross/clashcross /app/bin/clashcross
+
+      # Install the icon.
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      #cp -r assets/icons/$projectId.svg $iconDir/
+      - cp -r com.clashcross.clashcross.png /app/share/icons/hicolor/scalable/apps/
+
+      # Install the desktop file.
+      - mkdir -p /app/share/applications
+      #cp -r packaging/linux/$projectId.desktop $desktopFileDir/
+      - cp -r com.clashcross.clashcross.desktop /app/share/applications/
+
+      # Install the AppStream metadata file.
+      - mkdir -p /app/share/metainfo
+      - cp -r metainfo.xml /app/share/metainfo/
```

```suggestion
      - install -Dm644 metainfo.xml /app/share/metainfo/$FLATPAK_PID.metainfo.xml
```

it was installed with the wrong name.